community/php5-xdebug/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Andy Postnikov <apostnikov@gmail.com>
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Contributor: Fabio Ribeiro <fabiorphp@gmail.com>
+# Maintainer: Andy Postnikov <apostnikov@gmail.com>
+pkgname=php5-xdebug
+pkgver=2.5.1
+pkgrel=0
+_pkgver=${pkgver/_rc/RC}
+_phpver=${pkgname%%-*}
+_pkgreal=${pkgname#*-}
+pkgdesc="$_phpver extension: provides functions for function traces and profiling"
+url="http://pecl.php.net/package/$_pkgname"
+arch="all"
+license="PHP"
+pecldepends="$_phpver-dev autoconf"
```

Comment:
I don't like it too, but I puted this variable because old phpext-template has it.
So I'll just replace it in community/php7-* to "php7-dev automake".
For testing/php7-* I'll use php7-pecl instead of php7-dev, because php7-pecl contains all dependencies to build extension.